engine/glfw/lib/ios/app/AppDelegate.m

```diff
@@ -43,20 +43,6 @@ @implementation AppDelegate
 
 @synthesize window;
 
-- (void)forceDeviceOrientation;
```

Comment:
This is the reason of the bug.
It was added as a fix of[ the dev app on iOS 8.0](https://github.com/defold/defold/commit/29a654bfd30cf6ba991a3e5b4a954b3fd2d39258).
We don't have dev app anymore and user can create it's own or use the game as a dev app

[Later](https://github.com/defold/defold/commit/70e7531c93d7fd66657f5977e85c8c2e4c15a27e) it was moved as part of a workaround for iOS 8.0.0-8.0.2 which we don't support anymore. our min iOS version is 9.0